### PR TITLE
🌐 [iOS] Use error messages from Moko strings in tests

### PR DESCRIPTION
**Pull request by JuliaJakubcova**
# :pencil: Description
- Use error messages from Moko strings in tests

# :bulb: What’s new?
- Using MR strings in tests

# :no_mouth: What’s missing?
- Running tests on CI

### CODE REVIEW

File: ios/DomainLayer/SharedDomain/Sources/SharedDomain/StoreKit/UseCases/RequestFeedbackUseCase.swift [hunk 1]
```diff
@@ -19,12 +19,6 @@ public struct RequestFeedbackUseCaseImpl: RequestFeedbackUseCase {
     }
     
     public func execute() throws {
-        if Environment.flavor == .debug {
-            guard CommandLine.arguments.contains("-ShouldShowFeedback") else { return }
-            
-            try storeKitRepository.requestFeedback()
-        } else {
-            try storeKitRepository.requestFeedback()
-        }
```
Comment (pchmelar) [inline ios/DomainLayer/SharedDomain/Sources/SharedDomain/StoreKit/UseCases/RequestFeedbackUseCase.swift]: Why this change? Please discuss with @DavidKadlcek, as he introduced this code in https://github.com/MateeDevs/devstack-native-app/pull/99

Comment (JuliaJakubcova) [reply ios/DomainLayer/SharedDomain/Sources/SharedDomain/StoreKit/UseCases/RequestFeedbackUseCase.swift]: Ahhhh, it was crashing at this bit, ~~David already fixed it in his branch~~, I think, this was just to help me get to my tests and I forgot to remove this change, sorry

Comment (JuliaJakubcova) [reply ios/DomainLayer/SharedDomain/Sources/SharedDomain/StoreKit/UseCases/RequestFeedbackUseCase.swift]: I added @DavidKadlcek 's solution as a new commit, now all tests work 🙂

**Review by pchmelar — CHANGES_REQUESTED** [review]
Just one minor improvement/suggestion - please use `#warning("TODO: ...")` instead of `// TODO: ...` - it will automatically shows in Xcode warnings, so it's more likely we won't forget about it 🙂

**Comment by JuliaJakubcova** [comment]
> Just one minor improvement/suggestion - please use `#warning("TODO: ...")` instead of `// TODO: ...` - it will automatically shows in Xcode warnings, so it's more likely we won't forget about it 🙂

@pchmelar it's great that we can make it a warning, but it's sad that the link is no longer clickable and that it cannot be placed anywhere (for example it does not work here in the parameters of a function anymore)
Screenshot 2024-06-06 at 07 35 29

**Review by pchmelar — APPROVED** [review]
